apps/nocobase/1.4.13/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   nocobase:
-    image: nocobase/nocobase:1.4.13
+    image: nocobase/nocobase:1.4.14
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The changes made to the `nocobase` service configuration appear to be minor corrections. No other irregularities, issues, or optimizations are present based on the provided snippet at this time. The image has been updated from version 1.4.13 to 1.4.14, which may affect performance improvements and bug fixes since that release cycle. Consider running tests with the new image to verify stability.